minimal/pom.xml

```diff
@@ -106,7 +106,7 @@
                     <artifactId>maven-pmd-plugin</artifactId>
                     <configuration>
                         <excludeRoots>
-                            <excludeRoot>target/generated-sources/stubs</excludeRoot>
+                            <excludeRoot>target/generated-sources</excludeRoot>
```

Comment:
Most generated code should be excluded (because it usually does not pmd or findbugs muster). Adding this however means that even if code *should* be checked, it never is unless someone notices it. The current setting ensures that all code gets checked and is deactivated if desired on a case-by-case base. 

I would recommend having a setting like this in a corporate pom.